pagopa/pn-frontend
# PN-13440 - Rework IOContact

## Short description
Rework of IOContact component

## List of changes proposed in this pull request
- Update copy
- Add illustrations

## How to test
Start the PF app and log in. There are three possible scenarios:

1. AppIO Unavailable: you should see a button that redirects you to the AppIO landing page
2. AppIO Disabled: you should see a button that allows you to enable the AppIO service. When you click this button, a confirmation dialog should appear
3. AppIO Enabled: you should see a “Disattiva” button in the header of the card, which allows you to disable the AppIO service
